vividus-plugin-web-app-playwright/src/main/java/org/vividus/ui/web/playwright/steps/PageSteps.java

```diff
@@ -65,7 +65,6 @@ public void openMainApplicationPage()
     @Given("I am on page with URL `$pageUrl`")
     public void openPage(String pageUrl)
     {
-        uiContext.reset();
```

Comment:
why is it removed?

Reply:
uiContext.reset() has been removed here since it is called on the onFrameNavigated event.

Reply:
just to confirm: whenever we do any navigation, `onFrameNavigated` event is fired, right?

Reply:
yes, that's right